docs/shared/guides/environment-variables.md

```diff
@@ -83,3 +104,164 @@ To interpolate an environment variable named `NX_DOMAIN_NAME` into your `index.h
   </body>
 </html>
 ```
+
+### Using environment variables in Angular applications
+
+By default, Angular only provides the `NODE_ENV` variable when building the application. You may use `process.env.NODE_ENV`
+anywhere in your TS/JS source, and the build will inline this value in the output chunks.
+
+Other variables, such as those prefixed by `NX_` will not work in Angular. To add support for other environment variables,
+do the following.
+
+First, install `@angular-builders/custom-webpack` package so we can customize webpack, and also `@types/node` so we can
+use `process` in our code.
+
+```bash
+npm install --save-dev @angular-builders/custom-webpack @types/node
+
+# Or with yarn
+yarn add --dev @angular-builders/custom-webpack @types/node
+```
+
+Next, update the `build` and `serve` targets (in `project.json` or `angular.json` file), to the following.
+
+```json lines
+{
+  "build": {
+    // NOTE: change the executor to one that supports custom webpack config.
+    "executor": "@nrwl/angular:webpack-browser",
+    // snip
+    "options": {
+      // NOTE: This file needs to be created.
+      "customWebpackConfig": {
+        "path": "apps/myapp/webpack.config.js"
+      }
+      // snip
+    }
+  },
+  "serve": {
+    // NOTE: use dev-server that supports custom webpack config.
+    "executor": "@angular-builders/custom-webpack:dev-server"
```

Comment:
Our `@nrwl/angular:webpack-browser` and `@nrwl/angular:webpack-server` support custom webpack configuration. We should probably recommend using those instead.